core/src/main/scala/better/files/UnicodeCharset.scala

```diff
@@ -73,7 +75,9 @@ class BomEncoder(charset: Charset) extends CharsetEncoder(charset, 1, 1) {
     .toArray
   private[this] var isBomWritten = false
 
-  override def encodeLoop(in: CharBuffer, out: ByteBuffer): CoderResult = {
+  override def encodeLoop(_in: CharBuffer | Null, _out: ByteBuffer | Null): CoderResult = {
```

Comment:
**Overridden java method arguments are nullable**
A Scala method overrides some Java method. We need to add `|Null` to the method's parameter types to make the override still work.